__tests__/browserifySpec.js

```diff
@@ -20,7 +21,7 @@ describe('itemjs general tests', function() {
     actors: ['e']
   }]
 
-  var itemsjs = require('./../dist/itemsjs')(items);
```

Comment:
The idea of it was to test compiled itemsjs to see if it's working (compilation and minification is correct). Also it was increasing a chance that it will work for VanillaJs. That's why also this test file is called `browserifySpec.js`

Is there a chance to make similar checking in this environment ?

Reply:
Gotcha, let me look into this

Reply:
We should somehow find a way to test it out programatically here.

When I test movies with `https://unpkg.com/itemsjs@1.0.39/dist/itemsjs.min.js` here `https://jsfiddle.net/0ef9qeos/167/` it works correctly 

When I test `https://cdn.rawgit.com/saminzadeh/itemsjs/es6-nwb-build/dist/itemsjs.min.js` here `https://jsfiddle.net/51nvd4m0/` then I am getting errors.

This is how you said that it might be a problem of not correct compilation / minification

Reply:
The more I look into it, the more I'm thinking of reproaching this PR and scraping the big changes that aren't directly related to #12

The original motivation was to make sure the code compiled through all build systems (like create-react-app) and had good cross browser support. The only _real_ issue with the current master is that there are [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) which are not fully supported in all node/browser envs.

Instead, let's do the bare minimal to get this working with your current setup. That being:
- Add babel preprocessing to convert code to standard ES5 before its piped through browserify
- Publish the transpiled babel output in `lib/` to NPM on `npm publish`


Here is what that looks like on my personal branch:
https://github.com/saminzadeh/itemsjs/tree/babel-browserify

The following changes are not within scope. We can submit issues seperately to handle those later.
- New testing framework Jest
- Converting to ES6 (ES2015) imports
- Changing the build system from browserify to webpack (NWB)
- Cherry picking the used lodash modules at build time

That being said, I would recommend converting this codebase to ES6 at some point to be more consistent with modern JS development and enable a new set of tooling (eg. [Lodash babel plugin needs ES6 imports to work](https://github.com/lodash/babel-plugin-lodash#limitations)). It's difficult to convert the code to ES6 with auto tooling since `module.exports` are in various places.


Thanks for your help and willingness to allow me to contribute!

Reply:
I see, all of that makes sense! Please make a PR between development and https://github.com/saminzadeh/itemsjs/tree/babel-browserify and I'll merge it. 

In meantime I definitely need to learn more things about babel, react, etc to be up-to-date :) 

Thanks for your suggestions and providing modern ideas / approach!